lib/controller/messagescontroller.php

```diff
@@ -124,6 +130,7 @@ public function index($from=0, $to=20, $filter=null) {
 	/**
 	 * @NoAdminRequired
 	 * @NoCSRFRequired
+	 * @UseSession
```

Comment:
FIXME: Hacky workaround. Needs to get assigned earlier to allow closing session here.